system/jlib/jutil.cpp

```diff
@@ -3367,6 +3367,27 @@ extern jlib_decl offset_t friendlyStringToSize(const char *in)
     return result * scale;
 }
 
+extern jlib_decl double friendlyCPUToDecimal(const char *in)
```

Comment:
Should be commoned up with (probably replace) 
`getCpuSize()
` in workunit.cpp.

Reply:
ah, didn't look hard enough for a similar impl. - I thought there might be one somewhere.
I'll common up.